src/CMakeLists.txt

```diff
@@ -538,6 +538,7 @@ endif()
 
 add_subdirectory(kv)
 add_subdirectory(os)
+add_subdirectory(blkdrv)
```

Comment:
i'd suggest s/blkdrv/blk/

@neha-ojha @jdurgin @aclamk  what do you think?

Reply:
Please consider using the proper name to distinguish it from "src/blkin"
BTW, what should be the name of the new library?  Could we keep using libblkdrv?

Reply:
@tchaikov 
I prefer "block". "blk" is ok, "blkdrv" is bad.